### PR TITLE
Change anytype to unknown primitive

### PR DESCRIPTION
**Pull request by roironn**

### CODE REVIEW

File: packages/salesforce-adapter/src/constants.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ export enum FIELD_TYPE_NAMES {
   ROLLUP_SUMMARY = 'Summary',
   // internal-only placeholder for fields whose type is unknown
   UNKNOWN = 'Unknown',
+  ANYTYPE = 'AnyType'
```
Comment (netama) [inline packages/salesforce-adapter/src/constants.ts:55]: does https://github.com/salto-io/salto/blob/f5134f5db40daea3d94eba18501c88402de27201/packages/salesforce-adapter/src/constants.ts#L80 need to change?

Comment (roironn) [reply packages/salesforce-adapter/src/constants.ts:55]: :+111111

File: packages/salesforce-adapter/src/constants.ts [hunk 2]
```diff
@@ -77,7 +78,7 @@ export const COMPOUND_FIELDS_SOAP_TYPE_NAMES:
 
 export const FIELD_SOAP_TYPE_NAMES:
 Record<string, FIELD_TYPE_NAMES> = {
-  anyType: FIELD_TYPE_NAMES.TEXT, // TODO: define specific type
+  anyType: FIELD_TYPE_NAMES.ANYTYPE, // TODO: define specific type
```
Comment (ori-moisis) [inline packages/salesforce-adapter/src/constants.ts:81]: remove the TODO?

Comment (roironn) [reply packages/salesforce-adapter/src/constants.ts:81]: lol yes

**Review by ori-moisis — APPROVED** [review]

**Pull request merged**